src/_gettsim/transfers/unterhaltsvors.py

```diff
@@ -68,56 +107,73 @@ def unterhaltsvors_m(  # noqa: PLR0913
     if (
         out > 0
         and (alter >= unterhaltsvors_params["altersgrenze_mindesteinkommen"])
-        and (unterhaltsvorschuss_eink_m_tu < unterhaltsvors_params["mindesteinkommen"])
+        and (not _unterhaltsvorschuss_eink_above_income_threshold_fg)
     ):
         out = 0.0
 
-    # Check against the actual child alimony payments given by kindesunterhalt_m
-    out = max(out - kind_unterh_erhalt_m, 0.0)
-
     return out
 
 
-def unterhaltsvorschuss_eink_m_tu(  # noqa: PLR0913
-    bruttolohn_m_tu: float,
-    sonstig_eink_m_tu: float,
-    eink_selbst_m_tu: float,
-    eink_vermietung_m_tu: float,
-    kapitaleink_brutto_m_tu: float,
-    sum_ges_rente_priv_rente_m_tu: float,
-    arbeitsl_geld_m_tu: float,
+def _unterhaltsvorschuss_eink_above_income_threshold(
+    unterhaltsvorschuss_eink_m: float,
+    unterhaltsvors_params: dict,
+) -> bool:
+    """Check if income is above the threshold for advance alimony payments.
+
+    Parameters
+    ----------
+    unterhaltsvorschuss_eink_m
+        See :func:`unterhaltsvorschuss_eink_m`.
+    unterhaltsvors_params
+        See params documentation :ref:`unterhaltsvors_params <unterhaltsvors_params>`.
+
+    Returns
+    -------
+
+    """
+    return unterhaltsvorschuss_eink_m >= unterhaltsvors_params["mindesteinkommen"]
+
+
+def unterhaltsvorschuss_eink_m(  # noqa: PLR0913
+    bruttolohn_m: float,
+    sonstig_eink_m: float,
+    eink_selbst_m: float,
+    eink_vermietung_m: float,
+    kapitaleink_brutto_m: float,
+    sum_ges_rente_priv_rente_m: float,
+    arbeitsl_geld_m: float,
 ) -> float:
-    """Calculate relevant income for advance on alimony payment on tax unit level.
+    """Calculate relevant income for advance on alimony payment on Steuernummer level.
```

Comment:
```suggestion
    """Calculate relevant income for advance on alimony payment.
```
We're individual-level here, right?